f8a_worker/workers/init_analysis_flow.py

```diff
@@ -23,6 +23,7 @@ def execute(self, arguments):
         :param arguments: dictionary with task arguments
         :return: {}, results
         """
+        self.log.info("Arguments passed by UnknownDependencyFetcherTask: {}".format(arguments))
```

Comment:
In this specific case yes, but in general the arguments to `InitAnalysisFlow` are not passed from `UnknownDependencyFetcherTask`.

Also can we change `self.log.info()` to `self.log.debug()` everywhere you add these **debug** messages ?

To summarize, I'd expect something like
`self.log.debug("Input arguments: {}".format(arguments))`

Reply:
Okay sure :)